velox/exec/OutputBufferManager.cpp

```diff
@@ -135,7 +137,11 @@ bool OutputBufferManager::updateOutputBuffers(
 void OutputBufferManager::updateNumDrivers(
```

Comment:
Can we return a boolean indicating whether the update was processed or task no longer being tracked. This would allow the caller to take appropriate action and this would match the behavior of updateOutputBuffers API. CC: @xiaoxmeng

Reply:
We can do that.